packages/wrangler/src/config/validation.ts

```diff
@@ -2008,6 +2018,63 @@ const validateDurableObjectBinding: ValidatorFn = (
 	return isValid;
 };
 
+/**
+ * Check that the given field is a valid "workflow" binding object.
+ */
+const validateWorkflowBinding: ValidatorFn = (_diagnostics, _field, _value) => {
+	// if (typeof value !== "object" || value === null) {
```

Comment:
Can we just remove all these commented-out lines? Or, it seems like we just return true from this, so maybe we don't need to include this function at all?

Reply:
Argh I noticed this and meant to come back and add some validation

Reply:
i've deleted the commented code and explicitly returned true with a TODO comment